### PR TITLE
Translation of css chapter to Japanese

### PR DESCRIPTION
**Pull request by ksakae1216**
Please review for me.
Part of [#586](https://github.com/HTTPArchive/almanac.httparchive.org/issues/586)

I've also updated src/content/en/2022/css.md
Please confirm it too.

Which one should I choose about the project and milestones?

### CODE REVIEW

File: src/content/ja/2022/css.md [hunk 1]
```diff
@@ -0,0 +1,1347 @@
...
+}}
+
+2020年、2021年と同様、ウェブ上でもっとも普及しているクラス名は`active`です。Font Awesomeのプレフィックスである`fa`、`fa-*`は依然として2位と3位を占めています。しかし、`wp-*`クラス名は順位を上げ、4位となりました。2021年には20％だったのが、現在では31％のページで表示されています。また、`has-large-font-size`といったクラス名も登場しており、これらは新しいWordPressブロックエディターで使用されています。
+
+トップ20から`clearfix`が消え、今では10%のページで見られるだけです。これは、フロートベースのレイアウトがウェブから消えつつあることを明確に示しています。
+
+{{ figure_markup(
+    image="top-selector-ids.png",
+    caption="ページ数の割合でもっとも普及しているID名です。",
+    description="もっとも多くのページで使用されているCSS IDを示す棒グラフ。モバイルとデスクトップの傾向は似ています。IDコンテンツは15%のページで使用され、次にフッターが12%、ヘッダー、`fb-root`、`fb_dialog_loader_close`、`fb_dialog_ipad_overlay`、`fb_dialog_loader_spinner`が10%、レスポンスとコメントが9%、そして最後にメインが8%のページで使用されていることがわかります。",
```
Comment (tunetheweb) [inline src/content/ja/2022/css.md:111]: Could you double check this one as think you've translated some of the code words.

Here's the English version:

```
    description="Bar chart showing the CSS IDs used on the most pages. Mobile and desktop trends are similar. The ID `content` is used on 15% of pages, followed by `footer` on 12%, `header`, `fb-root`, `fb_dialog_loader_close`, `fb_dialog_ipad_overlay`, and `fb_dialog_loader_spinner` all on 10%, then `respond` and `comments` on 9%, and finally `main` on 8% of pages.",
```

File: src/content/ja/2022/css.md [hunk 2]
```diff
@@ -0,0 +1,1347 @@
...
+}}
+
+2021年に見られるように、大半のカスタムプロパティは深さが0であり、他のカスタムプロパティの値をその値に含めないことを意味する。深度1のプロパティの数は少し増え、深度2の数は少し減りました。しかし、このデータからは、過去1年間にカスタムプロパティの使い方がそれほど複雑になったとは思えません。
+
+## 色
+
+{{ figure_markup(
+    image="color-formats.png",
+    caption="もっとも普及しているカラーフォーマットの出現率。",
+    description="もっとも多くのページで使用されているカラーフォーマットを棒グラフで示したもの。6桁の`#RRGGBB`構文が49%のページ、3桁の`#RGB`が25%のページ、`rgba`関数が14%のページ、透明キーワードが8%のページ、名前付きカラーが2%、`rgb`関数は1%、残りのフォーマットは1%未満のページで使用されており、以下の通りであった。8桁の`#RRGGBBAA`、`hsla`関数、`currentColor`キーワード、4桁の`#RGBA`、システムカラーキーワード、`hsl`関数、カラー関数、`hwb`関数、`lch`関数、そして最後に`lab`関数が、人気順に並んでいます。",
```
Comment (tunetheweb) [inline src/content/ja/2022/css.md:587]: Similarly can you check this translation

Here's the English:

```
description="Bar chart showing the color formats used on the most pages. the six-digit `#RRGGBB` syntax is used on 49% of pages, three-digit `#RGB` on 25% of pages, the `rgba` function on 14% of pages, the `transparent` keyword on 8% of pages, a named color on 2%, the `rgb` function on 1%, and the rest of the formats were used on less than 1% of pages, including: eight-digit `#RRGGBBAA`, `hsla` function, `currentColor` keyword, four-digit `#RGBA`, system color keywords, `hsl` function, `color` function, `hwb` function, `lch` function, and finally the `lab` function, in descending order of popularity.",
```

File: src/content/ja/2022/css.md [hunk 3]
```diff
@@ -0,0 +1,1347 @@
...
+このグラフは興味深いもので、グラフの下限では、デスクトップとモバイルでほぼ同じサイズの画像を提供していること、50および75パーセンタイルでは、デスクトップよりもはるかに大きな画像をモバイルユーザーに提供していることを表しています。このデータからわかることは、モバイルユーザーに対して、より大きな画像を配信しているということです。おそらく、タブレット端末のランドスケープモードを考慮しているのでしょう。
+
+## レイアウト
+
+ウェブでレイアウトを行う際には、多くの選択肢から選ぶことができ、ほとんどのサイトでは、これらのさまざまな方法が使用されていることでしょう。使用されているレイアウト方法を検出するために、プロパティと値の組み合わせでデータを簡単に検索すると、次の表が得られます。
+
+{{ figure_markup(
+    image="layout-props.png",
+    caption="ページ数の割合によるレイアウト方法。",
+    description="もっとも多くのページで使用されているレイアウト方法を示した棒グラフです。ブロックとアブソリュートレイアウトが92%のページで使用され、次いでインラインブロックが90%、フロートは89%、フィックスでは84%、インラインが82%、CSSテーブルは79%、フレックスでは77%、ボックスが51%のページで使用されています。",
```
Comment (tunetheweb) [inline src/content/ja/2022/css.md:726]: And again double check:

```
description="Bar chart showing the layout methods used on the most pages. `block` and `absolute` layouts are used on 92% of pages, followed by `inline-block` on 90%, `floats` on 89%, `fixed` on 84%, `inline` on 82%, CSS tables on 79%, `flex` on 77%, and `box` on 51% of pages.",
```

File: src/content/ja/2022/css.md [hunk 4]
```diff
@@ -0,0 +1,1347 @@
...
+新しい `aspect-ratio` プロパティは、2%のページで使用されています。これは2021年末から相互運用可能になったので、このプロパティの使用率が時間とともに伸びていくのが興味深いところです。
+
+## トランジションとアニメーション
+
+`animation`プロパティは、モバイルページの77％（昨年と同じ）に表示され、デスクトップでは76.8％とわずかに増加しました。`transition` プロパティはさらに人気があり、モバイルページの85％、デスクトップページの85.6％に見受けられます。デスクトップの頻度は、2021年から4ポイントほどわずかに減少しています。
+
+{{ figure_markup(
+    image="transition-props.png",
+    caption="もっとも普及している `transition` プロパティを、ページ数の割合で表示します。",
+    description="もっとも多くのページで使用されているトランジションプロパティを棒グラフで表示しています。もっとも多くのページで使われているプロパティは`all`53％で、以下、`opacity`が50％、`transform` 38％、`none` 25％、`height` 22％、`color` 21％、`background-color` 20％、`background` 17％、`box-shadow` 13％、`left` 12％、`width`、`top`と `-webkit-transform` 10%、`border-color`と`visibility` 8％と続いています。",
```
Comment (tunetheweb) [inline src/content/ja/2022/css.md:830]: Can you check:

```
description="Bar chart showing the `transition` properties used on the most pages. The most popular property `all` is used on 53% of pages, followed by `opacity` on 50%, `transform` 38%, `none` 25%, `height` 22%, `color` 21%, `background-color` 20%, `background` 17%, `box-shadow` 13%, `left` 12%, `width`, `top`, and `-webkit-transform` 10%, `border-color` and `visibility` 8%.",
```

File: src/content/ja/2022/css.md [hunk 5]
```diff
@@ -0,0 +1,1347 @@
...
+    sql_file="transition_timing_functions.sql"
+  )
+}}
+
+2021年と比較すると、遷移時に使用されるタイミング機能の分布はほとんど変化していない。当時と同様に、明確なトップは`ease`です。
+
+{{ figure_markup(
+    image="transition-animation-names.png",
+    caption="アニメーション名で識別されるアニメーションの種類。",
+    description="もっとも多く使われているアニメーションの種類を示す棒グラフ。もっとも普及しているのはuncategorizedで13%、次いで`rotate` 13%、`bounce` 11%、`slide` 10%、`fade` 9%、`wobble` 5%、`scale` 4%、`pulse` 2%、`visibility` 2%となっています。",
```
Comment (tunetheweb) [inline src/content/ja/2022/css.md:903]: "uncategorized" is not a real value but a description. Can you translate?

```
description="Bar chart showing the animation types used most of the time. The most popular type is uncategorized at 13%, followed by `rotate` at 13%, `bounce` at 11%, `slide` at 10%, `fade` at 9%, `wobble` at 5%, `scale` at 4%, `pulse` at 2%, and `visibility` at 2%.",
```

File: src/content/ja/2022/css.md [hunk 6]
```diff
@@ -0,0 +1,1347 @@
...
+}}
+
+ブレンドモードでもっともよく見られる値は`multiply`で、42%のページで見られました。しかし、他の値についても、かなりの割合で分布しています。
+
+約18%のページが、カスタムプロパティ `var(--overlay-mix-blend-mode)` を使っていました。これは、何らかのライブラリやツールに由来する、特定の名前です。
+
+{{ figure_markup(
+    image="filter-functions.png",
+    caption="フィルターを設定するページでもっとも普及しているフィルター機能。",
+    description="フィルターを設定しているページでもっとも多く使われている関数を棒グラフにしたもの。アルファ関数が82％、フィルターなし（none）59％、`progid:DXImageTransform.Microsoft.gradient` 46％、`blur` 31％、`drop-shadow`と`grayscale` 22％、`brightness` 20％、`progid:DXImageTransform.Microsoft.BasicImage`と`inherit` 10％、`url`、`sepia`、`contrast`、`none` `!important`すべて8％となっています。",
```
Comment (tunetheweb) [inline src/content/ja/2022/css.md:946]: Are we missing alpha in the translation?

```
description="Bar chart showing the functions used on the most pages that set filters. The `alpha` function is used 82% of the time, followed by no filter (none) 59%, `progid:DXImageTransform.Microsoft.gradient` 46%, `blur` 31%, `drop-shadow` and `grayscale` 22%, `brightness` 20%, `progid:DXImageTransform.Microsoft.BasicImage` 16%, `saturate` 11%, `inherit` 10%, `url`, `sepia`, `contrast`, and `none !important` all 8%.",
```

File: src/content/ja/2022/css.md [hunk 7]
```diff
@@ -0,0 +1,1347 @@
...
+    sql_file="media_query_features.sql"
+  )
+}}
+
+`prefers-*` ユーザープリファレンス機能だけを見ると、`prefers-reduced-motion` がもっとも普及していることがわかります。これは、ブラウザのサポートが良く、ウェブ上でアニメーションや遷移が普及していることが原因です。`prefers-color-scheme` 機能は、ユーザーが明るい色か暗い色のどちらを好むかをチェックする機能で、ウェブサイトやアプリケーションでダークモードを使用することが一般的になったため、使用率がわずかに増加しています。
+
+{{ figure_markup(
+    image="hover-features.png",
+    caption="ホバーやポインターのメディア機能を利用する。",
+    description="ホバーとポインターのメディア機能の人気を示す棒グラフ。ホバー機能は5%、ポインターでは2%、エニーポインターとエニーホバーは1%以下のページで使用されています。",
```
Comment (tunetheweb) [inline src/content/ja/2022/css.md:1006]: Can you check this one for code formating?

```
description="Bar chart showing the popularity of hover and pointer media features. The `hover` feature is used on 5% of pages, `pointer` 2%, and `any-pointer` and `any-hover` below 1% of pages.",
```

File: src/content/ja/2022/css.md [hunk 8]
```diff
@@ -0,0 +1,1347 @@
...
+繰り返しますが、ブレイクポイントは圧倒的にピクセル値で設定されているので、他の値をピクセルに変換してグラフにすることはしていません。最初の `em` 値は、やはり `48em` で、位置78で見つかりました。
+
+## クエリで変更されたプロパティ
+
+メディアクエリブロック内に表示されるプロパティを調べ、ブレークポイントに応じてどのプロパティを変更しているかを確認しました。
+
+{{ figure_markup(
+    image="media-query-props.png",
+    caption="メディアクエリーブロックでもっとも普及しているプロパティです。",
+    description="メディアクエリブロックの中でもっとも多くのページで使用されているプロパティを示す棒グラフ。ディスプレイと幅のプロパティは83%のページでメディアクエリに含まれており、`height`と`padding`は78%、`margin-left` 77%、`font-size` 76%、`margin`と`position` 75%、`margin-right`、`left`、`top`、`margin-top`、`max-width`  74%、`right`と`margin-bottom` 73%、`padding-left` 72%、`text-align` 71%、`padding-right` 70%、`background` 69%、`float` 67%と続いています。",
```
Comment (tunetheweb) [inline src/content/ja/2022/css.md:1040]: Not sure we should translate `display` and `width`?

```
description="Bar chart showing the properties found in media query blocks that are used on the most pages. The `display` and `width` properties are found in media queries on 83% of pages, followed by `height` and `padding` on 78%, `margin-left` 77%, `font-size` 76%, `margin` and `position` 75%, margin-right, `left`, `top`, `margin-top`, and `max-width` 74%, `right` and `margin-bottom` 73%, `padding-left` 72%, `text-align` 71%, `padding-right` 70%, `background` 69%, and `float` 67%.",
```

File: src/content/ja/2022/css.md [hunk 9]
```diff
@@ -0,0 +1,1347 @@
...
+[Houdini](https://developer.mozilla.org/ja/docs/Web/CSS/CSS_Houdini)は、オープンなWeb上ではまだほとんど使われていないんです。アニメーションのカスタムプロパティを使用しているページ数を見ると、2021年以降、わずかな増加にとどまっています。また、Houdini Paint APIの利用状況も調べてみました。私たちは、Web上でこれが使用されている事例を発見しました。使用されている小作品名を見ると、その多くが <a hreflang="en" href="https://css-houdini.rocks/smooth-corners/">smooth corners</a> 小作品で、通常の `border-radius` にうまくフォールバックできることから、人々はこれを段階的強化として使用していることがわかります。
+
+## Sass
+
+Sassのようなプリプロセッサーは、開発者がCSSでできるようになりたいと思っていてもできないことの良い指標と見ることができます。そしてCSSのパワーアップに伴い、開発者からのよくある質問は、Sassを使う必要がまったくないのか、というものです。カスタムプロパティの利用が増加していることから、プリプロセッサでよく使われる変数や定数の機能が、CSSに組み込まれ同等の機能を持つようになったことがわかります。
+
+{{ figure_markup(
+    image="sass-function-calls.png",
+    caption="もっとも普及しているSass関数呼び出しの割合。",
+    description="もっとも多く使われているSass関数呼び出しを示した棒グラフ。Sass関数呼び出しのうち、`if`関数が19％を占め、以下、その他の関数17％、`darken` 14％、`map-get` 10％、`map-keys` 9％、パーセンテージ6％、`nth` 5％、`lighten`と`mix` 4％、`type-of`、α調整、単位、長さが3％と続きます。",
```
Comment (tunetheweb) [inline src/content/ja/2022/css.md:1151]: `percentage`, `unit` and `length` shouldn't be translated:

```
description="Bar chart showing the Sass function calls that used most of the time. The `if` function makes up 19% of Sass function calls, followed by other functions 17%, `darken` 14%, `map-get` 10%, `map-keys` 9%, `percentage` 6%, `nth` 5%, `lighten` and `mix` 4%, `type-of`, alpha adjustment, `unit`, and `length` 3%.",
```

**Review by tunetheweb — CHANGES_REQUESTED** [review]
Looks great!

However it appears we didn't have code formatting in a lot of the descriptions in English version. I've added those to this PR, but need you to double check some of the translations as think you might have translated some of the code.

**Review by ksakae1216 — COMMENTED** [review]
@tunetheweb  Fix one line

File: src/content/ja/2022/css.md (hunk 1 above)
Comment (tunetheweb) [reply src/content/ja/2022/css.md:111]: So why does `footer` and `header` not show as code words in your translation?

Comment (ksakae1216) [reply src/content/ja/2022/css.md]: @tunetheweb  Thanks I translated

**Review by tunetheweb — APPROVED** [review]

**Pull request merged**